### PR TITLE
MDCT-2337: Non-Required Measurement Fix (SPIKE Output)

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

In a few instances, state users reported that the auto-completed (i.e. read-only) measures were being set to `In Progress` even though these should always be marked `Complete`. After taking a look with @BearHanded we saw that on a session time-out, the auto-completed measures are getting "saved" and thus returning to the `In Progress` state.

### Related ticket(s)

MDCT-2337

---
### How to test

- Navigate to any FY Adult Measures
- Verify that `NCIDDS-AD` is set to `Complete` by default
- Navigate away and back
- It should still be set `Complete`

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by BearHanded — CHANGES_REQUESTED** [review]
I think this bug would still break a normally completed measure if the user timed out while looking at it

File: services/ui-src/src/components/MeasureWrapper/index.tsx [hunk 1]
```diff
@@ -224,7 +225,10 @@ export const MeasureWrapper = ({
   };
 
   const handleSave = (data: any) => {
-    if (!mutationRunning && !loadingData) {
+    // do not auto-save measure if the measure has already been completed or if it an auto-complete/read-only measure
+    const shouldSave =
+      measureStatus === MeasureStatus.INCOMPLETE || !autoCompletedMeasure;
+    if (!mutationRunning && !loadingData && !shouldSave) {
```
Comment (benmartin-coforma) [inline services/ui-src/src/components/MeasureWrapper/index.tsx:231]: I think this condition is backwards? Unless I'm misunderstanding, we want `shouldSave` and not `!shouldSave`.

Comment (karla-vm) [reply services/ui-src/src/components/MeasureWrapper/index.tsx:231]: That is correct, just fixed it! thanks 🏆

**Review by benmartin-coforma — DISMISSED** [review]
Looks good!

**Review by BearHanded — APPROVED** [review]

**Review by keeysnc — APPROVED** [review]

**Pull request merged**